lib/mandriller/version.rb

```diff
@@ -1,3 +1,3 @@
 module Mandriller
-  VERSION = '0.2.0'
+  VERSION = '0.2.1'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.